build.gradle

```diff
@@ -240,6 +243,15 @@ dependencies {
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
   }
 
+  functionalTestCompile sourceSets.test.runtimeClasspath
+  functionalTestAnnotationProcessor 'org.projectlombok:lombok:1.18.12'
```

Comment:
One more dependency would be required functionalTestCompileOnly 'org.projectlombok:lombok:1.18.12'

Reply:
dependency added